noir-projects/noir-protocol-circuits/crates/types/src/constants.nr

```diff
@@ -410,134 +432,89 @@ global START_EMIT_UNENCRYPTED_LOG_WRITE_OFFSET: u32 = START_EMIT_L2_TO_L1_MSG_WR
  * GAS COSTS FOR AVM OPCODES
  * They are used in TS and C++
  */
-global AVM_DEFAULT_BASE_GAS_L2: u16 = 100;
-global AVM_DEFAULT_DYN_GAS_L2: u16 = 0;
-// Base GAS.
-global AVM_ADD_BASE_L2_GAS: u16 = 320;
-global AVM_SUB_BASE_L2_GAS: u16 = 320;
-global AVM_MUL_BASE_L2_GAS: u16 = 330;
```

Comment:
Every line is changed here because I had to make them u32s so I could do math with the other constants defined above as u32s (tree heights, gas constants). I tried making those u16s, but that opened a whole can of worms since u32 is sprinkled across the protocol circuits.